packages/account/src/account.test.ts

```diff
@@ -7,33 +7,24 @@ import { ASSET_A, ASSET_B } from '@fuel-ts/utils/test-utils';
 
 import type { FakeResources, TransferParams } from './account';
 import { Account } from './account';
-import { FUEL_NETWORK_URL } from './configs';
+import type { CoinQuantity, Resource } from './providers';
 import { ScriptTransactionRequest, Provider } from './providers';
 import * as providersMod from './providers';
-import type { CoinQuantity, Resource } from './providers';
-import { generateTestWallet, seedTestWallet } from './test-utils';
+import { TestAssetId, setupTestProviderAndWallets } from './test-utils';
 import { Wallet } from './wallet';
-
 /**
  * @group node
  */
 
 describe('Account', () => {
-  const assets = [ASSET_A, ASSET_B];
-  let provider: Provider;
-  let baseAssetId: string;
-
-  beforeAll(async () => {
-    provider = await Provider.create(FUEL_NETWORK_URL);
-    baseAssetId = provider.getBaseAssetId();
-    assets.push(baseAssetId);
-  });
-
   afterEach(() => {
     vi.restoreAllMocks();
   });
 
-  it('should create account using an address, with a provider', () => {
+  it('should create account using an address, with a provider', async () => {
+    using launched = await setupTestProviderAndWallets();
```

Comment:
A lot of test changes follow this pattern:
```ts
using launched = await setupTestProviderAndWallets();

const { provider } = launched;
```

I suggest you create a test helper function:
```ts
export function setupTestProvider(providerOptions: ProviderOptions) {
    const {provider, cleanup} = await setupTestProviderAndWallets({providerOptions});

    return Object.assign(provider, {[Symbol.dispose]: cleanup})
}
```
Then you don't have to do the destructuring:
```ts
using provider = await setupTestProvider();
```

Reply:
As much as I agree with reducing repetition, abstraction should ideally be avoided in tests. We could also use the insights here for potential DX improvements on the Test Node Launcher instead of delegating them for the consumers to abstract away.

Reply:
The helper can still be included in this PR, as it is simple enough.

I just want to raise awareness about the possibility that our end users will probably repeat themselves just like us. To remedy that, we may also consider potential improvements from the utility perspective.

Reply:
f818d8106